cpp/ql/src/change-notes/2024-07-23-using-enum-declaration.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
```

Comment:
Apologies. I missed initially that you put this in `cpp/ql/src/change-notes`. Since this is a library change, this needs to go in `cpp/ql/lib/change-notes`. Moving it to that location means we can also do the following, which makes more sense:
```suggestion
category: feature
```

Reply:
Ah, that's a good point. Yeah, `minorAnalysis` indeed seems better.